spec/requests/api/merge_requests_spec.rb

```diff
@@ -358,14 +361,27 @@
 
   describe "POST /projects/:id/merge_request/:merge_request_id/comments" do
     it "should return comment" do
-      post api("/projects/#{project.id}/merge_request/#{merge_request.id}/comments", user), note: "My comment"
-      expect(response.status).to eq(201)
-      expect(json_response['note']).to eq('My comment')
+      post api(merge_request_comments_url, user), note: 'My comment'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.